channels-rke2.yaml

```diff
@@ -14,6 +14,8 @@ appDefaults:
         defaultVersion: '1.25.x'
       - appVersion: '>= 2.7.5-0 < 2.7.100-0'
         defaultVersion: '1.26.x'
+      - appVersion: '>= 2.7.5-0 < 2.7.100-0'
```

Comment:
I think we'd need to revise min/max channel server version and this range depending on target milestone